src/requests/getImages.js

```diff
@@ -0,0 +1,27 @@
+import axios from "axios";
+
+const getImages = (query) => {
+  if (!query) {
+    return Promise.resolve([]);
+  } else {
+    return axios
+      .get(`https://images-api.nasa.gov/search?q=${query}`)
+      .then((res) => {
+        const imageResults = res.data.collection.items;
+        const parsedImages = imageResults.filter((image) => {
+          return image.data[0].media_type === "image";
+        });
+
+        const images = parsedImages.map((image) => {
+          return image.links[0].href;
+        });
+
+        return images;
```

Comment:
This could condensed a little to avoid creating a variable that only gets used once:
```suggestion
        return parsedImages.map((image) => {
          return image.links[0].href;
        });
```

Reply:
Code condensed.